hpx/runtime/actions/continuation.hpp

```diff
@@ -124,12 +124,12 @@ namespace hpx
         }
     }
 
-    template <typename T>
-    void set_lco_value(naming::id_type const& id, naming::address && addr, T && t,
+    template <typename Result, typename RemoteResult>
+    void set_lco_value(naming::id_type const& id, naming::address && addr, RemoteResult && t,
```

Comment:
Do we really need two template arguments now? Couldn't we use `traits::promise_local_result` instead of explicitly specifying `Result`?